fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/LaoActivity.java

```diff
@@ -68,6 +66,8 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
     witnessingViewModel = obtainWitnessingViewModel(this, laoId);
 
+    obtainPoPCHAViewModel(this, laoId).disableConnectingFlag();
```

Comment:
why `disableConnectingFlag` ? It's somewhat surprising to read this here, as someone who's somewhat unfamiliar with the codebase.

Reply:
Indeed, double checking I think it's unnecessary because when switching LAO the view model is recreated and the initial state of the flag is already false :)